roles/openshift_logging/tasks/scale.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- shell: >
+    {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig get {{object}}
+    --template='{{ '{{.spec.replicas}}' }}' -n {{openshift_logging_namespace}}
+  register: replica_count
+  failed_when: replica_count.rc == 1 and 'exists' not in replica_count.stderr
+  when: not ansible_check_mode
+
+- shell: >
+    {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig scale {{object}}
+    --replicas={{desired}} -n {{openshift_logging_namespace}}
+  register: scale_result
+  failed_when: scale_result.rc == 1 and 'exists' not in scale_result.stderr
+  when:
+  - replica_count.stdout != desired
+  - not ansible_check_mode
+
+- shell: >
+    {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig describe {{object}} -n {{openshift_logging_namespace}} | awk -v statusrx='Pods Status:' '$0 ~ statusrx {print $3}'
```

Comment:
If i use jsonpath i can pull this value from `'{.status.availableReplicas}'` i'm updating to use that instead